src/app/routes/enrolment/view-requests/issuer-requests/issuer-requests.component.ts

```diff
@@ -84,7 +84,7 @@ export class IssuerRequestsComponent
   }
 
   updateExpirationDate(expirationTime: number): void {
-    this.expirationTime = expirationTime;
+    this.expirationTime = Date.now() + expirationTime * 1000;
```

Comment:
I would change this param to `validityPeriod` as I think it is more accurate. 

I think this overall situation is a bit confusing though as the `expirationTime`/`validityPeriod` param is in milliseconds but `this.expirationTime` is in seconds? Maybe it's better to do the `* 1000` before this function so that everything is in seconds for this function?

```suggestion
    this.expirationTime = Date.now() + validityPeriod * 1000;
```

Reply:
I updated the param name here, but @dawidgil  I'll let you weigh in on where we should do the multiplication

Reply:
I think the multiplication should be done in `app-expiration-date` component. At least you can adjust calculation there :)

Reply:
I don't think expirationTime is being passed into that component to do the calculation there? From what I see it stays in issuer-request?